### PR TITLE
use ripunzip for Windows to unzip faster

### PR DESCRIPTION
**Pull request by adrpo**

### CODE REVIEW

File: OMCompiler/Compiler/Script/CevalScript.mo [hunk 1]
```diff
@@ -2964,14 +2964,18 @@ protected function unZipEncryptedPackageAndCheckFile
...
 algorithm
   success := false;
   outFilename := "";
   if (System.regularFileExists(filename)) then
     if (StringUtil.endsWith(filename, ".mol")) then
       workdir := if System.directoryExists(inWorkdir) then inWorkdir else System.pwd();
-      if (skipUnzip or 0 == System.systemCall("unzip -q -o -d \"" + workdir + "\" \"" +  filename + "\"")) then
+      // use ripunzip on Windows as is twice as fast
+      cmdPrefix := if isWindows then "ripunzip.exe -q unzip-file -d " else "unzip -q -o -d ";
+      cmd := cmd + "\"" + workdir + "\" \"" +  filename + "\"";
```
Comment (adeas31) [inline OMCompiler/Compiler/Script/CevalScript.mo:2977]: Its suppose to be `cmd := cmdPrefix+ "\"" + workdir + "\" \"" + filename + "\"";`

**Review by adeas31 — CHANGES_REQUESTED** [review]
Can `ripunzip` unzip to a specific directory?

**Comment by adeas31** [comment]
Just saw your msg that it has `-d` flag :)

File: OMCompiler/Compiler/Script/CevalScript.mo (hunk 1 above)
Comment (adrpo) [reply OMCompiler/Compiler/Script/CevalScript.mo:2977]: ouch :)

**Comment by adeas31** [comment]
Maybe we should also mention from where `ripunzip` comes from so we don't forget.

**Review by adeas31 — APPROVED** [review]

**Pull request merged**